### ISSUE TITLE
Provide feedback to user on JVM heap memory usage

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Illablu/google-refine#467]
```
As discussed in the comments for issue 147, better feedback to users about 
memory usage is desirable.  Ideally, we should warn them in advance when they 
attempt an operation which has no hope of succeeding due to too little memory, 
but future memory requirements are difficult to predict.

One possible improvement would be to provide real-time updates on memory usage 
as part of the progress indicator during project creation.  This would help the 
user identify when they are using almost all of the memory and progress is 
grinding to a halt.
```

Original issue reported on code.google.com by `tfmorris` on 21 Oct 2011 at 4:25
- Blocked on: #607

**Comment by GoogleCodeExporter** [comment]
```
In r2341 I've added a first cut at this which will show the used JVM heap vs 
total available as part of the create project progress indicator.
```

Original comment by `tfmorris` on 21 Oct 2011 at 5:48
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
With the improvements from issue 607, I'm going to consider the initial pass 
complete.

We could probably do more in terms of flushing cached projects when we're low 
on memory, but that can be a separate task.
```

Original comment by `tfmorris` on 8 Sep 2012 at 6:37
- Changed state: **Fixed**
- Now blocked on: #607

**Comment by GoogleCodeExporter** [comment]
Original comment by `tfmorris` on 18 Sep 2012 at 3:05
- Added labels: **Milestone-2.6**
- Now blocked on: #607
